<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RightWidget</class>
 <widget class="QTabWidget" name="RightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>572</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>572</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab_historyList">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>330</width>
     <height>546</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <attribute name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/rightImage/music.png</normaloff>:/rightImage/music.png</iconset>
   </attribute>
   <attribute name="title">
    <string>播放列表</string>
   </attribute>
   <attribute name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;播放列表&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListWidget" name="listWid_histMusic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>544</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_favor">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>330</width>
     <height>546</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <attribute name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/rightImage/disklike.png</normaloff>:/rightImage/disklike.png</iconset>
   </attribute>
   <attribute name="title">
    <string>我喜欢</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="tab_native">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>330</width>
     <height>546</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <attribute name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/bottomImage/listLoop.png</normaloff>:/bottomImage/listLoop.png</iconset>
   </attribute>
   <attribute name="title">
    <string>本地音乐</string>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
